packages/oasis-actions/src/strategies/ajna/open.ts

```diff
@@ -0,0 +1,78 @@
+import BigNumber from 'bignumber.js'
+import * as ethers from 'ethers'
+
+import ajnaProxyActionsAbi from '../../abi/ajna/ajnaProxyActions.json'
+import { AjnaPosition } from '../../helpers/ajna'
+import { Address, Strategy } from '../../types/common'
+import * as views from '../../views'
+
+export interface OpenArgs {
+  poolAddress: Address
+  dpmProxyAddress: Address
+  quoteAmount: BigNumber
```

Comment:
I think we use a `debt` across our app.

Reply:
Please fight with @piotrkonowrocki he asked me to align it with Ajna naming.

Reply:
Was going to say same thing

Reply:
Let's agree on that and then then I'll apply changes to final version